dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/support/AbortPolicyWithReport.java

```diff
@@ -134,4 +138,17 @@ private void dumpJStack() {
 
     }
 
+    private String getDumpPath() {
+        final String dumpPath = url.getParameter(DUMP_DIRECTORY);
+        if (StringUtils.isEmpty(dumpPath)) {
+            return USER_HOME;
+        }
+        final File dumpDirectory = new File(dumpPath);
+        if (!dumpDirectory.exists() && !dumpDirectory.mkdirs()) {

```

Comment:
@zenuo maybe we need warn users if we create the folders ?

Reply:
Sounds like a good idea, how about make a `info` log? Because creating a folder may be a routine operation.